zorm/src/main/kotlin/com/zenmo/orm/companysurvey/SurveyRepository.kt

```diff
@@ -91,6 +91,22 @@ class SurveyRepository(
         }
     }
 
+    fun setActive(surveyId: UUID, userId: UUID, active: Boolean) {
+        val nUpdated = transaction(db) {
```

Comment:
it shouldn't be in a try-catch to handle more potential errors?

Reply:
I think it's ok to let them bubble up.